pkg/scheduler/plugins/coscheduling/controller/podgroup.go

```diff
@@ -225,13 +225,15 @@ func (ctrl *PodGroupController) syncHandler(key string) error {
 			return err
 		}
 		pods = append(pods, podFromInformer)
+		klog.Infof("pgController get pod from informer : %v", podFromInformer.Name)
```

Comment:
use proper log level. for more details, please check https://github.com/koordinator-sh/koordinator/blob/main/docs/contributors/logging.md